packages/kokoas-client/src/pages/projContractV2/sections/hasProjPlan/IncludePlanContractAmtRadio.tsx

```diff
@@ -0,0 +1,60 @@
+import { Controller } from 'react-hook-form';
+import { useTypedFormContext } from '../../hooks/useTypedRHF';
+import { FormControl, FormControlLabel, Radio, RadioGroup, Typography } from '@mui/material';
+import { Stack } from '@mui/system';
+
+export const IncludeContractPlanAmt = () => {
+  const { control } = useTypedFormContext();
+   
+  return (
+    <Stack 
+      direction={'row'}
+      spacing={2}
+      alignItems={'center'}
+    >
+      <Typography fontSize={14}>
+        「設計契約金」は含まれますか？
```

Comment:
```suggestion
        この契約書には「設計契約金」は含まれますか？
```

対象が分かりづらいかと思いましたので。
依頼者との間で合意は取れていると思うので、修正はお任せします

Reply:
ありがとうございます！そのまま適用させて頂きます。